### PR TITLE
Features section

### PR DESCRIPTION
**Pull request by lcordoba**
# Summary

- Implements features section

# Trello Card

https://trello.com/c/4dRR7qZt/2-maquetar-la-seccion-de-features-de-la-pagina

# Screenchots

captura de pantalla de 2017-07-17 12-36-15

captura de pantalla de 2017-07-17 12-37-25

captura de pantalla de 2017-07-17 12-37-41

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,5 +11,25 @@
       <div class="divider"></div>
       <img class="logo" src="images/tcg_logo.png" alt="Logo">
     </div>
+    <div class="triangles-div"></div>
+    <section>
+      <div class="section features">
```
Comment (JesusHerrera) [inline index.html]: why are you using a "section" class?

File: css/style.css [hunk 2]
```diff
@@ -44,6 +44,41 @@ header {
...
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+p {
```
Comment (JesusHerrera) [inline css/style.css]: Try keeping tags and class selectors separated.

File: index.html (hunk 1 above)
Comment (lcordoba) [inline index.html]: Fixed.
@JesusHerrera

File: css/style.css (hunk 2 above)
Comment (lcordoba) [inline css/style.css]: Fixed.
@JesusHerrera

**Pull request merged**